en/02_Developer_Guides/15_Customising_the_Admin_Interface/07_ReactJS_Redux_and_GraphQL.md

```diff
@@ -1249,25 +1430,64 @@ const mutation = {
   templateName: CREATE,
   singularName: 'Note',
   pagination: false,
-  params: {},
+  params: {
+    input: 'CreateNoteInput!', // For graphql v3 use 'AppNoteCreateInputType!'
+  },
   fields: [
-    'Content',
-    'ID'
+    'content',
+    'id'
   ],
 };
 
 export default mutation;
 ```
 
-It looks like a lot of code, but if you're familiar with Apollo mutations, this is pretty standard. The supplied `mutate` function gets mapped to a prop -- in this case `onAdd`, which the `AddForm` component is configured to invoke. We've also supplied the `singularName` as well as the template `CREATE` for the `createNote` scaffolded mutation.
+[notice]
+With graphql v3, the field names in the input object have to start with capital letters, and the input _type_ can't easily be overridden - it's always the first part of your namespace, then the class name, then "CreateInputType". So assuming your model's fully qualified classname is `App\Model\Note`, the input type is `AppNoteCreateInputType`.
+[/notice]
+
+It looks like a lot of code, but if you're familiar with Apollo mutations, this is pretty standard. The supplied `mutate()` function gets mapped to a prop -- in this case `onAdd`, which the `AddForm` component is configured to invoke. We've also supplied the `singularName` as well as the template `CREATE` for the `createNote` scaffolded mutation.
```

Comment:
```suggestion
It looks like a lot of code, but if you're familiar with Apollo mutations, this is pretty standard. The supplied `mutate()` function gets mapped to a prop — in this case `onAdd`, which the `AddForm` component is configured to invoke. We've also supplied the `singularName` as well as the template `CREATE` for the `createNote` scaffolded mutation.
```

Reply:
m-dash

Reply:
there seems to be multiple occurrences in the document as well

Reply:
Yeah those were pre-existing (I've moved some of them but didn't make new ones) - but I'll update them now that you're pointing them out :p

Reply:
done

Reply:
@michalkleiner, do you have any other comments or can I merge this PR?
Thank you in advance.